go/libkb/chain_link.go

```diff
@@ -474,11 +607,16 @@ func ComputeLinkID(d []byte) LinkID {
 	return LinkID(h[:])
 }
 
-func (c *ChainLink) VerifyHash() error {
+func (c *ChainLink) verifyHash() error {
 	if c.hashVerified {
 		return nil
 	}
 
+	// For V2 Sigchain links, we might not have the payload (to save bandwidth, etc)
+	if c.IsStubbed() {
+		return nil
```

Comment:
Is it safe to short-circuit here? Would it be safe to hoist responsibility for skipping this check onto the caller?